app/helpers/finance_helper.rb

```diff
@@ -50,4 +50,12 @@ def latest_induction_record_for_provider?(induction_record, participant_profile)
 
     induction_record == participant_profile.latest_induction_record_for(cpd_lead_provider:)
   end
+
+  def authorise_for_payment_button_visible?(statement)
```

Comment:
ah I think one last thing to add is hiding the button if `mark_as_paid_at` is set but still payable (after a job is triggered but pending)

Reply:
All done @cwrw I hope it's all good.. tks for your review..